ls_1/index.html

```diff
@@ -0,0 +1,529 @@
+            <img src="/images/prefer-pic4.svg" width="198" alt="prefer pic" />
+            <p>Graphic Design</p>
+          </div>
+          <img src="/images/slide-right.svg" width="24" alt="Slide right" />
+        </div>
+      </section>
+      <section class="customers">
+        <img
+          src="/images/two-women-img.svg"
+          width="412"
+          alt="Two women image"
+        />
+        <div class="customers__feedback">
+          <h2>Our great customers<span> _____</span></h2>
+          <img
+            src="/images/customers-feedback-img.svg"
+            width="39"
+            alt="Customers feedback image"
+          />
+          <p>
+            Sed ut perspiciatis unde omnis iste natus error sit<br />
+            voluptatem accusantium doloremque
+          </p>
+          <p>
+            Laudantium, totam rem aperiam, eaque ipsa quae ab illo<br />
+            inventore veritatis et quasi architecto beatae vitae dicta<br />
+            sunt explicabo.
+          </p>
+          <h3>Stephanie Magion</h3>
+          <img src="/images/customers-scroll.svg" width="70" alt="Scroll" />
+        </div>
+      </section>
+      <section class="ready">
+        <p>IT'S FAST, FREE AND VERY EASY!</p>
+        <article>
+          <h2>Ready to get started?</h2>
+          <p>
+            Sed ut perspiciatis unde omnis iste natus error sit<br />
+            voluptatem accusantium doloremque
+          </p>
+          <p>
+            Laudantium, totam rem aperiam, eaque ipsa quae ab illo<br />
+            inventore veritatis et quasi architecto beatae vitae dicta<br />
+            sunt explicabo.
+          </p>
+          <div>
+            <a href="#form">Apply Now</a>
+            <button>Request Service</button>
+          </div>
+        </article>
+        <aside>
+          <div>
+            <img src="/images/link.svg" width="24" alt="Link" />
+            <h2>Quick Link</h2>
+          </div>
+          <div>
+            <img src="/images/pen-tool.svg" width="24" alt="Pen" />
+            <p>Tution and Fees</p>
+            <hr />
+          </div>
+          <div>
+            <img src="/images/case.svg" width="24" alt="Case" />
+            <p>Review & Rating</p>
+            <hr />
+          </div>
+          <div>
+            <img src="/images/book-open.svg" width="24" alt="Book" />
+            <p>Universoty Facilities</p>
+            <hr />
+          </div>
+          <div>
+            <img src="/images/comment.svg" width="24" alt="Comment" />
+            <p>Community Q&A</p>
+            <hr />
+          </div>
+        </aside>
+      </section>
+      <section class="blog">
+        <h2>Latest Blog<span> _____</span></h2>
+        <div class="blog__container">
+          <div class="blog__container--item">
+            <img src="/images/blog-img1.svg" width="415" alt="Blog image" />
+            <div class="blog__container--itemDesign">
+              <p>D</p>
```

Comment:
Have a look on

text-orientation: upright;
writing-mode: vertical-rl;

Reply:
Thanks, changed to one <p>